### PR TITLE
[EN] asg: rewrite

### PR DESCRIPTION
**Pull request by MegaApplePi**
updated

### CODE REVIEW

File: wiki/Article_Style_Guide/en.md [hunk 1]
```diff
@@ -642,95 +483,110 @@ or
...
+Whenever linking to a beatmap, use this format as the link text:
 
 ```
-{artist} - {title} ({creator}) \[{difficuty_name}\]
+{artist} - {title} ({creator}) [{difficuty_name}]
 ```
 
 When linking to a beatmap, make sure that the link actually links to that difficulty.
-All beatmap difficulty urls looks like this: `https://new.ppy.sh/b/{BeatmapID}`
+All beatmap difficulty URLs looks like this: `https://new.ppy.sh/b/{BeatmapID}`.
```
Comment (EpicSupermind) [inline wiki/Article_Style_Guide/en.md]: shouldn't it be https://osu.ppy.sh, not new.ppy.sh?

File: wiki/Article_Style_Guide/en.md [hunk 2]
```diff
@@ -642,95 +483,110 @@ or
...
 
-If you are linking to a beatmapset, use this format as the link text:
+Whenever linking to a beatmapset, use this format as the link text:
 
 ```
 {artist} - {title} ({creator})
 ```
 
-All beatmapset difficulty URLs looks like this: `https://new.ppy.sh/s/{BeatmapSetID}`
+All beatmapset difficulty URLs looks like this: `https://new.ppy.sh/s/{BeatmapSetID}`.
```
Comment (EpicSupermind) [inline wiki/Article_Style_Guide/en.md]: `https://osu.ppy.sh/s/{BeatmapSetID}`

File: wiki/Article_Style_Guide/en.md [hunk 3]
```diff
@@ -642,95 +483,110 @@ or
...
+- the image is not inside a heading
+- the image has a title/hover text
 
-### Image Links
+The syntax for captions are the same as normal Markdown images.
 
-Do **not** give the images links.
+#### Image Links
+
+**Images are never to be given links!**
```
Comment (EpicSupermind) [inline wiki/Article_Style_Guide/en.md]: `**Images are never to be in links!**`
idk

File: wiki/Article_Style_Guide/en.md [hunk 4]
```diff
@@ -764,11 +632,52 @@ Some tools for beautifying tables are:
 - [Markdown Table Formatter](http://markdowntable.com/)
```
Comment (EpicSupermind) [inline wiki/Article_Style_Guide/en.md]: https:// ?

File: wiki/Article_Style_Guide/en.md [hunk 5]
```diff
@@ -1,627 +1,468 @@
...
-However, do not expect a lot out of Markdown as it is intended to be used as a bare minimum markup language.
-
-### Syntax
-
-If you need help with GFM syntax, see [markdown-cheatsheet-online.pdf](https://guides.github.com/pdfs/markdown-cheatsheet-online.pdf) or [Markdown Cheatsheet](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet).
-
-If you really need to, you could also ask for some help in the [osu!dev discord](https://discord.gg/ppy).
-
-## Folder Names
+The items listed in this article only applies to the `wiki/` directory.
```
Comment (kj415j45) [inline wiki/Article_Style_Guide/en.md]: Unused define

File: wiki/Article_Style_Guide/en.md [hunk 6]
```diff
@@ -1,627 +1,468 @@
...
+**Examples:**
+
 - `` `.jpg` ``
 - `` `.exe` ``
 - `` `.osz` ``
 
-### Chat Channels
+#### Chat Channels
 
 When copying the name of a chat channel, start it with a hash (`#`), followed by the channel name in lower case letters.
```
Comment (kj415j45) [inline wiki/Article_Style_Guide/en.md]: redefine at L700

File: wiki/Article_Style_Guide/en.md (hunk 4 above)
Comment (MegaApplePi) [inline wiki/Article_Style_Guide/en.md]: Please, take the time to make sure https works before commenting about it. And no, this site does not support https.

File: wiki/Article_Style_Guide/en.md (hunk 5 above)
Comment (MegaApplePi) [inline wiki/Article_Style_Guide/en.md]: please explain

File: wiki/Article_Style_Guide/en.md (hunk 6 above)
Comment (MegaApplePi) [inline wiki/Article_Style_Guide/en.md]: please explain

File: wiki/Article_Style_Guide/en.md [hunk 7]
```diff
@@ -1,627 +1,468 @@
...
-- if a section starts with a space, ignore it
+**Examples:**
 
-Notice: you will not be able to check if section link links to the correct location in some scenarios.
+- `[ASG](/wiki/ASG)`
+- `[OWC2015](/wiki/OWC2015)`
+- `[skinning tutorial](/wiki/skinning_tutorial)`
 
-Examples:
+##### Why are we using redirects instead of paths?
```
Comment (MegaApplePi) [inline wiki/Article_Style_Guide/en.md]: Please explain why this change should happen. I will not change it unless you provide a good explanation of some sort.

File: wiki/Article_Style_Guide/en.md [hunk 8]
```diff
@@ -1,627 +1,468 @@
...
+| `zh-tw.md` | Chinese (Taiwan, Traditional) | 中文(繁體) |
+
+## Repository Directory
+
+These rules applies to the folders of the `osu-wiki` repository.
 
-All folders must **never** contain URL reserved characters.
+### Folder Names
 
+**Folders must never contain URL reserved characters.**
```
Comment (sr229) [inline wiki/Article_Style_Guide/en.md]: L47: Overemphasized. Only Emphasize the important words in the context

File: wiki/Article_Style_Guide/en.md [hunk 9]
```diff
@@ -1,627 +1,468 @@
...
 
-or
+Index pages must link to their own subfolders and could contain descriptions of what each link links to.
 
-```
-# title of page
+## Markdown
 
-## heading level 2
+**These rules are to be followed for all articles.**
```
Comment (sr229) [inline wiki/Article_Style_Guide/en.md]: L70: Overemphasized. Only Emphasize the important words in the context

File: wiki/Article_Style_Guide/en.md (hunk 5 above)
Comment (kj415j45) [inline wiki/Article_Style_Guide/en.md]: in following section, you use `apply to` again and again

File: wiki/Article_Style_Guide/en.md (hunk 6 above)
Comment (kj415j45) [inline wiki/Article_Style_Guide/en.md]: L700 's meaning is the same as L372

Comment (MegaApplePi) [inline wiki/Article_Style_Guide/en.md]: They're slightly different though. Line 372 talks about chat channel names which are prefixed with a hash followed by their name in lowercase letters while line 700 talks about language names in how they must start with a capital letter.

File: wiki/Article_Style_Guide/en.md (hunk 8 above)
Comment (MegaApplePi) [inline wiki/Article_Style_Guide/en.md]: the whole sentence is important; not use **never**

File: wiki/Article_Style_Guide/en.md (hunk 9 above)
Comment (MegaApplePi) [inline wiki/Article_Style_Guide/en.md]: will unbold the entire thing, since line 9 isn't bold at all

File: wiki/Article_Style_Guide/en.md (hunk 8 above)
Comment (sr229) [inline wiki/Article_Style_Guide/en.md]: Not true. you only need to emphasize everything after "Folders must [..]"

Comment (MegaApplePi) [inline wiki/Article_Style_Guide/en.md]: 🤦‍♂️ fine

File: wiki/Article_Style_Guide/en.md (hunk 7 above)
Comment (sr229) [inline wiki/Article_Style_Guide/en.md]: This is a section though, keep it simple enough. Just put a reason why we use redirects instead of paths

Comment (MegaApplePi) [inline wiki/Article_Style_Guide/en.md]: changing to a horizontal bar

**Comment by kj415j45** [comment]
L41: please use 繁體中文 as tw's Native Script, this change will PR to osu-web later

I forgot it :p

**Comment by MegaApplePi** [comment]
@kj415j45 🆗

**Comment by kj415j45** [comment]
:tada: #347 will start!

**Pull request merged**